Sources/TuistSupport/Logging/FileLogger.swift

```diff
@@ -0,0 +1,30 @@
+import Foundation
+import TSCBasic
+
+public struct FileLogger: TextOutputStream {
+    enum FileHandlerOutputStream: Error {
+        case couldNotCreateFile
+    }
+
+    private let fileHandle: FileHandle
+    let encoding: String.Encoding
+
+    public init(path: AbsolutePath, encoding: String.Encoding = .utf8) throws {
+        if !FileManager.default.fileExists(atPath: path.url.path) {
+            guard FileManager.default.createFile(atPath: path.url.path, contents: nil, attributes: nil) else {
```

Comment:
Why not combine this into a single `guard - let statement`?

Reply:
Also, any reason why we're not using our `FileHandler` instead?

Reply:
Not really. I'll adjust it.